subsys/bluetooth/mesh/sensor.c

```diff
@@ -785,6 +789,11 @@ bool bt_mesh_sensor_value_in_column(const struct bt_mesh_sensor_value *value,
 void sensor_cadence_update(struct bt_mesh_sensor *sensor,
 			   const sensor_value_type *value)
 {
+	if (!sensor->state.configured) {
+		/* Ignore any update before cadence is configured. */
```

Comment:
Shouldn't `LOG_DBG` be added with appropriate message instead?
Here and above?

Reply:
No, because this is expected to happen. This is called for every publication. For publications happening before any cadence is configured (which is a normal thing to happen) we should ignore the cadence update call. I can of course move the ifs to all the call sites instead, but I found it easier to just add the check here and keep the call.